consensus/src/config.rs

```diff
@@ -113,8 +113,26 @@ pub fn default_config(n_members: NodeCount, node_ix: NodeIndex, session_id: Sess
     }
 }
 
+fn time_to_reach_round(round: Round, delay_schedule: DelaySchedule) -> Duration {
+    let mut total_time = Duration::from_millis(0);
+    for r in 0..round {
+        total_time += delay_schedule(r as usize);
+    }
+    total_time
+}
+
+/// Reaching a high round can be expected to introduce some slowdown.
+/// Use this function to ensure that the slowdown will happen.
+pub fn reaching_round_takes_at_least(
```

Comment:
This is fine, but I would suggest also adding something like `reaching_max_round_takes_at_least(&self, duration: Duration)` to `Config`, to make checking this even more convenient for users.